dependency-suppression/cve-suppressed.xml

```diff
@@ -5,6 +5,7 @@
    file name: preliquibase-spring-boot-starter-1.4.0.jar
    file name: preliquibase-spring-boot-autoconfigure-1.4.0.jar
    ]]></notes>
+        <filePath regex="true">.*\/preliquibase-spring-boot-starter-1\.4\.0\.jar\/.*\/preliquibase-spring-boot-autoconfigure-1\.4\.0\.jar</filePath>
```

Comment:
Can we do something like * for the versions?
This CVE was present in 1.3.x and now in 1.4.x as well
This CVE should not stop us from bumping the versions